src/include/duckdb/execution/operator/schema/physical_create_index.hpp

```diff
@@ -37,13 +37,31 @@ class PhysicalCreateIndex : public PhysicalOperator {
 	vector<unique_ptr<Expression>> expressions;
 	//! Info for index creation
 	unique_ptr<CreateIndexInfo> info;
-	//! Unbinded expressions to be used in the optimizer
+	//! Unbound expressions to be used in the optimizer
 	vector<unique_ptr<Expression>> unbound_expressions;
 
 public:
 	// Source interface
 	unique_ptr<GlobalSourceState> GetGlobalSourceState(ClientContext &context) const override;
```

Comment:
This should no longer be a source operator, and therefore no longer implement the source interface, right?

Reply:
Yes, I removed this originally, but then the pipeline complained that the root operator does not emit a chunk (is not a source), because this is the root operator in the plan. So I added this dummy function that just passes on the chunk, I'd love some help on how to fix this. :)

Reply:
Just do this analogous to `physical_vacuum`.